### DISCUSSION TITLE
GNS Server Summary window

### DISCUSSION THREAD

**Discussion by brianuhm** [GNS3/gns3-gui#3337] [General]
Before anyone says it, yes it was stupid : P 

I popped the server summary out to put it on a second monitor and accidentally x'ed out of it. I have dug online and in the ini files, yet I don't see anything that speaks to the server and topology windows. Is there anything that I can do short of re-installing?

Thanks in advance!

**Comment by ghost** [comment]
You can re-enable a closed dock with the menu option View -> Docks -> \<docking window\>. Since GNS3 v2.2.32 you can reset the dock configuration to the initial state by View -> Reset docks.

![gns3_enable_dock](https://user-images.githubusercontent.com/49040325/172632973-0a3c1b2c-7776-4d83-9c4b-f55cb83d2272.png)
